testcases/kernel/syscalls/utimensat/utimensat01.c

```diff
@@ -105,7 +108,9 @@ static void usageError(char *progName)
 	exit(EXIT_bad_usage);
 }
 
-int main(int argc, char *argv[])
+/* Renamed existing main to new_main for processing arguments */
+
+int new_main(int argc, char *argv[])
```

Comment:
this is really confusing.

Reply:
More comments added as to why this is renamed to new_main